stacker/commands/stacker/info.py

```diff
@@ -0,0 +1,15 @@
+"""Gets information on the CloudFormation stacks based on the given config."""
+
+from .base import StackerCommand
+from ...actions import info
+
+
+class Info(StackerCommand):
```

Comment:
Yea, I just wasn't sure if we wanted to put more information in their like
current status or something
On Sat, Aug 15, 2015 at 3:55 PM Michael Barrett <notifications@github.com>
wrote:

> In stacker/commands/stacker/info.py
> <https://github.com/remind101/stacker/pull/73#discussion_r37140292>:
>
> > @@ -0,0 +1,15 @@
> > +"""Gets information on the CloudFormation stacks based on the given config."""
> > +
> > +from .base import StackerCommand
> > +from ...actions import info
> > +
> > +
> > +class Info(StackerCommand):
>
> Does Info sound right to you? Maybe just Outputs for now or something like
> that?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/remind101/stacker/pull/73/files#r37140292>.
>